clover_api.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require_relative "./routes/base"
+
+class CloverApi < Roda
+  Unreloader.require "./routes/base.rb"
+
+  include CloverBase
+
+  plugin :default_headers,
+    "Content-Type" => "application/json"
+
+  plugin :hash_branches
+  plugin :json
+  plugin :json_parser
+
+  autoload_routes("api")
+
+  plugin :not_found do
+    {
+      error: {
+        code: 404,
+        title: "Resource not found",
+        message: "Sorry, we couldn’t find the resource you’re looking for."
+      }
+    }.to_json
+  end
+
+  plugin :error_handler do |e|
+    error = parse_error(e)
+
+    {error: error}.to_json
+  end
+
+  plugin :rodauth do
+    # Use JWT token for now, think more about authentication.
```

Comment:
```suggestion
```

I think the question of fancier authentication will be on our minds enough that we don't need a tiny comment to remind us. Mostly the upshot of such a design would be signing the content of the request and checking signatures of responses, which does oblige clients to be somewhat more complex. We may have to scope that and other more-stringent authentication stuff to particular projects, in practice I've found the ratio of development/staging projects to production projects is high...and only the production projects can justify significantly more annoyance and paranoia. It'd be nice if we were cognizant of this and avoided the need for a second identity (e.g. myname+prod@example.com), something I've done quite often.

Reply:
I think we can develop project based some features